src/services/air.js

```diff
@@ -1,85 +1,148 @@
-const GOOD = { text: 'Good', color: '#00CC00' };
-const MODERATE = { text: 'Moderate', color: '#F8B52A' };
-const UNHEALTHY_SENSITIVE = { text: 'Unhealthy for Sensitive Groups', color: '#EB8A14' };
-const UNHEALTHY = { text: 'Unhealthy', color: '#FF0000' };
-const VERY_UNHEALTHY = { text: 'Very Unhealthy', color: '#A10649' };
-const HAZARDOUS = { text: 'Hazardous', color: '#7E0023' };
-const NO_DATA = { text: 'No Data', color: '#757575' };
+const GOOD = { text: "Good", color: "#00e400", textColor: "#000" };
+const MODERATE = { text: "Moderate", color: "#ffff00", textColor: "#000" };
+const UNHEALTHY_SENSITIVE = {
+  text: "Unhealthy for Sensitive Groups",
+  color: "#ff7e00",
+  textColor: "#000",
+};
+const UNHEALTHY = { text: "Unhealthy", color: "#ff0000", textColor: "#000" };
+const VERY_UNHEALTHY = {
+  text: "Very Unhealthy",
+  color: "#8f3f97",
+  textColor: "#fff",
+};
+const HAZARDOUS = { text: "Hazardous", color: "#7e0023", textColor: "#fff" };
+const NO_DATA = { text: "No Data", color: "#757575", textColor: "#fff" };
+const PM_DATA = { text: "PM Level", color: "#757575", textColor: "#fff" };
+
+export const aqiColors = [
+  "#00e400",
+  "#ffff00",
+  "#ff7e00",
+  "#ff0000",
+  "#8f3f97",
+  "#7e0023",
+];
+
+export const aqiRanges = [
+  "0 - 50",
+  "51-100",
+  "101-150",
+  "151-200",
+  "201-300",
+  "301 - 500+",
```

Comment:
Odd mix of spacing between numbers and dashes here.

Also I think the `+` can go here as well. Basing off https://www.epa.gov/outdoor-air-quality-data/air-data-basic-information.

Reply:
Good catch. I've codified